### PR TITLE
seo connector

### LINKED ISSUE #2211: Local thread/Seo connector

### PR DESCRIPTION
**Pull request by brianosaurus**
Closes #2211 

Turns off indexeddb and uses in memory db
Runs the sync process in the UI thread

### CODE REVIEW

File: packages/augur-sdk/src/connector/seo-connector.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import { API } from "../state/api/API";
+import { Augur } from "../Augur";
+import { BlockAndLogStreamerListener } from "../state//db/BlockAndLogStreamerListener";
+import { Connector, Callback } from "./connector";
+import { ContractDependenciesEthers } from "contract-dependencies-ethers";
+import { Controller } from "../state/Controller";
+import { EthersProvider } from "@augurproject/ethersjs-provider";
+import { EventLogDBRouter } from "../state//db/EventLogDBRouter";
+import { JsonRpcProvider } from "ethers/providers";
+import { PouchDBFactory } from "../state//db/AbstractDB";
```
Comment (robert-e-davidson3) [inline packages/augur-sdk/src/connector/seo-connector.ts:11]: two slashes?

Comment (brianosaurus) [reply packages/augur-sdk/src/connector/seo-connector.ts:11]: good eye

File: yarn.lock [hunk 2]
```diff
@@ -16255,7 +16255,7 @@ pouchdb-adapter-leveldb-core@7.0.0:
     sublevel-pouchdb "7.0.0"
     through2 "2.0.3"
 
-pouchdb-adapter-memory@7.0.0:
+pouchdb-adapter-memory@7.0.0, pouchdb-adapter-memory@^7.0.0:
```
Comment (robert-e-davidson3) [inline yarn.lock]: I'm not sure what's going on here but it looks wrong to me.

Comment (brianosaurus) [reply yarn.lock]: good eye again.

Comment (robert-e-davidson3) [reply yarn.lock]: thanks!

**Review by robert-e-davidson3 — APPROVED** [review]
lgtm

**Pull request merged**